### ISSUE TITLE
SolrUtils: indexing of external files

### ISSUE THREAD

**Issue by invenio-developers** [inveniosoftware/invenio#1302]
Originally by _[pglauner](https://github.com/pglauner)_ on 2013-02-13

Solr shall be able to index external files which are not linked to a record through `BibRecDoc` for both BibIndex and BibRank.

The current workflow in BibIndex needs to be amended since a record might contain multiple external files in addition to BibRecDocs.

**Comment by hachreak** [comment]
it's still a issue for invenio v2.2?

**Comment by egabancho** [comment]
I just realise that I fixed this issue for legacy https://github.com/inveniosoftware/invenio/pull/3358 :open_mouth:

**Comment by egabancho** [comment]
I don't think this applies anymore to 2.2, BibRank are BibIndex don't exist on 2.2

@jalavik should we put back the milestone to 1.3?
